src/components/App.js

```diff
@@ -1,7 +1,49 @@
-import React from 'react'
+import React, { useState } from 'react'
+import Button from 'components/Button/Button'
+import Container from 'components/Container/Container'
+import Input from 'components/Input/Input'
+import List from 'components/List/List'
+import ListItem from 'components/ListItem/ListItem'
 
 const App = () => {
-  return <></>
+  const [items, setItems] = useState([])
+  const [todo, setTodo] = useState('')
+
+  const handleKeyDown = (event) => {
+    if (event.keyCode === 13 && event.target.value !== '') {
+      setItems([...items, todo])
+      addTodo()
+    }
+  }
+
+  const addTodo = () => setTodo('')
+
+  const handleAddTodoButtonClick = () => {
+    setItems([...items, todo])
+    addTodo()
+  }
+
+  const handleChange = (event) => setTodo(event.target.value)
+
+  return (
+    <>
+      <Container>
+        <Input
+          type="text"
+          placeholder="Ingrese nuevo To Do"
+          value={todo}
+          onKeyDown={handleKeyDown}
+          onChange={handleChange}
+        />
+        <Button onClick={handleAddTodoButtonClick}>Agregar</Button>
+        <List>
+          {items.map((item, index) => (
+            <ListItem key={index}>{item}</ListItem>
+          ))}
```

Comment:
Ahora tenemos todos, así que hay que cambiar las variables

```suggestion
          {todos.map((todo, index) => (
            <ListItem key={index}>{todo}</ListItem>
          ))}
```